adapter/internal/oasparser/model/mgw_swagger_dummy.go

```diff
@@ -0,0 +1,25 @@
+/*
+ *  Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Let's update the year
```suggestion
 *  Copyright (c) 2023, WSO2 LLC. (http://www.wso2.org) All Rights Reserved.
```